hdfs/src/main/scala/akka/stream/alpakka/hdfs/HdfsFlowStage.scala

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.hdfs
```

Comment:
It would be great to move internal stuff into an `impl` package. That would improve Java Module readiness.